src/pages/scientificServices/pipelines/views/RunJob.tsx

```diff
@@ -285,7 +304,13 @@ export const RunJob = () => {
                     </div>
                   )}
                   <ButtonPrimary
-                    disabled={!selectedPipeline || isSubmitting || !areAllRequiredInputsFilled() || !meetsMinimumQuota}
+                    disabled={
```

Comment:
this seems to be getting pretty crazy.  Is this normal in UI work like this or are we asking for the UI to be doing too much?

Reply:
fairly normal for components with a bunch of state, but your comment nudged me to take another look and I was able to eliminate `!selectedPipeline` as a condition because we don't display the button until a pipeline is selected